setup.py

```diff
@@ -1,11 +1,32 @@
-from setuptools import setup
+try:
+    from setuptools import setup, find_packages
+except ImportError:
+    from distutils.core import setup, find_packages
+
+description = (
+    'Codemod is a tool/library to assist you with large-scale codebase '
+    'refactors that can be partially automated but still require human '
+    'oversight and occasional intervention. Codemod was developed at '
+    'Facebook and released as open source.'
+)
 
 setup(
     name='codemod',
-    version='0.1',
+    version="0.2.1",
     url='http://github.com/facebook/codemod',
-    scripts=['src/codemod.py'],
-    author='facebook',
-    description='Codemod is a tool/library to assist you with large-scale codebase refactors that can be partially automated but still require human oversight and occasional intervention. Codemod was developed at Facebook and released as open source.',
-    zip_safe=True
+    license='Apache License 2.0',
+    author="Facebook",
+    author_email="facebook@facebook.com",
```

Comment:
Ha! Well, this seems like as good an email address as any, but I hope no one wastes their time sending anything to this address! :stuck_out_tongue_closed_eyes: